README.md

```diff
@@ -164,3 +164,14 @@ implement the `ViewResizer` protocol and resize yourself.
 The custom behavier can be used to record the views too.
 
 For more info on usage, check the [dynamic sizes tests](Bootstrap/BootstrapTests/DynamicSizeTests.swift).
+
+## A note about test failures artifacts
+
+By default Nimble-Snapshot will place test failures image diffs in a folder right next to `ReferenceImages`. It is strongly recommended that you don't commit those files to your source repo.
+
+Add this to your `.gitignore`:
+
+```
+# Nimble-Snapshot
+FailureDiffs
+```
```

Comment:
maybe this should be even more in evidence? I'd put it as a subitem of `Use`, before `Dynamic Type`